src/gitmoji.js

```diff
@@ -88,7 +91,13 @@ class GitmojiCli {
     return this._fetchEmojis()
       .then((gitmojis) => prompts.gitmoji(gitmojis))
       .then((questions) => {
+        if (config.getSignedCommit() !== constants.NONE) {
```

Comment:
You don't need to make this, the default value has to be set on the constructor, then you can remove completely the NONE constant.